File: 1012/1012 - Área.cpp
#include <iostream>
#include <iomanip>
int main() {
    double a, b, c, pi;
    pi = 3.14159;
    scanf("%lf%*c%lf%*c%lf", &a, &b, &c);
    printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n"
    , (a*c)/2, pi*(c*c), ((a+b)*c)/2.0, b*b, a*b);
    return 0;
}

File: 1012/1012 - Área.ex
[a, b, c] = IO.read(:stdio,:line) |> String.trim |> String.split(" ") |> Enum.map(fn str ->
  case Float.parse(str) do
    {number, _} -> number
  end
end)

pi = 3.14159

triangulo = (a * c) / 2 |> :erlang.float_to_binary([decimals: 3])
circulo = (pi * :math.pow c, 2) |> :erlang.float_to_binary([decimals: 3])
trapezio = ((a + b) * c) / 2 |> :erlang.float_to_binary([decimals: 3])
quadrado = (:math.pow b, 2) |> :erlang.float_to_binary([decimals: 3])
retangulo = a * b |> :erlang.float_to_binary([decimals: 3])

IO.puts "TRIANGULO: #{triangulo}\nCIRCULO: #{circulo}\nTRAPEZIO: #{trapezio}\nQUADRADO: #{quadrado}\nRETANGULO: #{retangulo}"
